nano/node/active_transactions.hpp

```diff
@@ -76,6 +77,13 @@ class confirmed_set_info final
 	nano::qualified_root root;
 };
 
+class dropped_election final
```

Comment:
This has the same definition as `confirmed_set_info`. An option is to rename that to `election_timepoint` or something like that and have only one class (and split into two classes if the types need to diverge)

This would allow the two multi_index_container members to be replaced with a type alias.

Reply:
Great suggestion, thanks!  29cd26b